harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/fragment/FragmentHarvester.java

```diff
@@ -351,19 +362,25 @@ private void createOrUpdateSubtemplate(Element fragment) throws Exception {
 
         String schema = fragment.getAttributeValue("schema");
 
+        String title = fragment.getAttributeValue("title");
```

Comment:
Hi Simon, can't we use the _title field for subtemplate which is defined by the XSL process ? https://github.com/geonetwork/core-geonetwork/blob/develop/schemas/iso19139/src/main/plugin/iso19139/index-fields/index-subtemplate.xsl#L83. The field could be using the ```@uuid``` attribute if needed ?

Reply:
@fxprunayre - finally got some time to do the indexing of the title from repo with _title in index-subtemplate.xsl - apologies for the delay. Have also made the handling of forward: consistent with the original forward: urls in MetadataApi - though I feel there must be a more sophisticated way of doing that in the labyrinth of spring somewhere... Should be ready for a final review. Thanks!